artifactory/commands/transferfiles/transferfileprogress.go

```diff
@@ -226,13 +224,8 @@ func (t *TransferProgressMng) RemoveRepository() {
 
 func (t *TransferProgressMng) changeNumberOfFailuresBy(n int) {
 	if t.ShouldDisplay() {
-		old := t.errorBar.GetTotal()
 		diff := int64(n)
-		if old+diff < 0 {
-			t.errorBar.SetGeneralProgressTotal(0)
-		} else {
-			t.errorBar.SetGeneralProgressTotal(old + diff)
-		}
+		t.errorBar.SetGeneralProgressTotal(t.errorBar.GetTotal() + diff)
```

Comment:
Isn't it possible that the diff will be negative?

Reply:
Well. if we get a negative value here, this means we have a bug, and we should avoid hiding it by "fixing" the result. Do you agree @omerzi?

Reply:
Agree 😄